### PR TITLE
Konstantinos/DPROD-3579/Accumulators Rest of the sections of options page

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

Revamp accumulators page first half

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/options/accumulator/accumulators_faq.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.accordion_border{
+    border-bottom: 2px solid #F6F7F8 ;
+}
```
Comment (hasan-deriv) [inline src/features/pages/options/accumulator/accumulators_faq.module.scss:3]: Can't we use tailwind for it? and why are we not using token for color name?

File: src/features/components/organisms/what-are-digital-options/index.tsx [hunk 2]
```diff
@@ -1,42 +1,97 @@
...
-                <Label text="_t_Available on Demo accounts only_t_" bgcolor="blue" />
-            )}
-            <Typography.Heading align="center" as="h1">
-                <Localize translate_text={heading} />
-            </Typography.Heading>
-            <Typography.Paragraph align="center">
-                <Localize translate_text={description} />
-            </Typography.Paragraph>
-        </Flex.Box>
+        <div className="max-w-[816px] mx-auto px-800">
```
Comment (hasan-deriv) [inline src/features/components/organisms/what-are-digital-options/index.tsx:40]: Why do we need extra div?

File: src/features/pages/options/accumulator/index.tsx [hunk 3]
```diff
@@ -1,42 +1,146 @@
...
+                <div id="faqs">
+                    <AccordionBlock
+                        className="border-opacity-black-100 border-x-none"
+                        content={{
+                            data: [
+                                [
+                                    {
+                                        className: `${styles.accordion_border}`,
+                                        content: function noRefCheck() {
+                                            return (
```
Comment (hasan-deriv) [inline src/features/pages/options/accumulator/index.tsx:36]: Why the function name is `noRefCheck`? Just curious

File: src/features/pages/options/accumulator/index.tsx [hunk 4]
```diff
@@ -1,42 +1,146 @@
...
+                    title={
+                        <Localize translate_text="_t_Why trade accumulator options on Deriv_t_" />
+                    }
+                    description={
+                        <Localize translate_text="Looking for a fast-paced, strategic and potentially explosive way to trade?" />
+                    }
+                    cols="three"
+                    cards={accumulatorCards}
+                />
+                <div id="faqs">
```
Comment (hasan-deriv) [inline src/features/pages/options/accumulator/index.tsx:27]: Why this `div` and `id` needed?

File: src/features/components/organisms/what-are-digital-options/index.tsx (hunk 2 above)
Comment (konstantinosG-deriv) [reply src/features/components/organisms/what-are-digital-options/index.tsx:40]: True, this is not needed. Removed it

File: src/features/pages/options/accumulator/accumulators_faq.module.scss (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/features/pages/options/accumulator/accumulators_faq.module.scss:3]: Will use token for color name but for the tailwind I tried putting it in the component from quill many times and it's not working. After a conversation with Meenu I will put it as scss for now until we fix it

File: src/features/pages/options/accumulator/index.tsx (hunk 4 above)
Comment (konstantinosG-deriv) [reply src/features/pages/options/accumulator/index.tsx:27]: Needed in order to scroll to the specific section when clicking the "Read FAQs" button. If I put the id in the AccordionBlock it doesn't scroll to the section

File: src/features/pages/options/accumulator/index.tsx (hunk 3 above)
Comment (konstantinosG-deriv) [reply src/features/pages/options/accumulator/index.tsx:36]: This function name is the default name from the AccordionBlock Quill component

**Review by hasan-deriv — APPROVED** [review]

**Review by dan-deriv — APPROVED** [review]

**Pull request closed without merging**